src/uris/deepLinks/deepLinkService.ts

```diff
@@ -19,15 +24,16 @@ import {
 } from './deepLink';
 
 export class DeepLinkService implements Disposable {
-	private _disposable: Disposable;
+	private _onDeepLinkEvent: Disposable | undefined;
```

Comment:
Since we don't actually care about this individually, I'd suggest just a `private readonly _disposables: Disposable[];`

Reply:
Done in commit: [Improvements from feedback](https://github.com/gitkraken/vscode-gitlens/pull/2503/commits/9e01ae826ec49b794057aa9570ec3df54ef0042d)